spec/system/log_in_system_spec.rb

```diff
@@ -6,4 +6,19 @@
       expect(page.find("h1")).to have_content "Dashboard"
     end
   end
-end
\ No newline at end of file
+
+  describe "Deactivated user" do
+    before do
+      create(:user, :deactivated, email: "deactivated@exmaple.com")
+    end
+
+    it "should not allow the user to log in" do
+      visit "/users/sign_in"
+      fill_in "user_email", with: "deactivated@example.com"
+      fill_in "user_password", with: "password"
+      find('input[name="commit"]').click
+
+      expect(page).to have_content("Invalid Email or password")
+    end
+  end
```

Comment:
@edwinthinks Let me know what you think of this test!

Reply:
Looks great! I think a future issue we can display a better error message which says that they've been deactivated.